addons/product_expiry/views/stock_quant_views.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding='UTF-8'?>
-<odoo>
-        <record id="view_quant_form_expiry" model="ir.ui.view">
-            <field name="name">stock.quant.inherit.form</field>
-            <field name="model">stock.quant</field>
-            <field name="inherit_id" ref="stock.view_stock_quant_form"/>
-            <field name="arch" type="xml">
-                <xpath expr="//field[@name='lot_id']" position="after" >
-                    <field name="removal_date"/>
-                </xpath>
-            </field>
-        </record>
-</odoo>
```

Comment:
explain in the commit message the rationale of why we remove this, ie the form view of the quant is never reachable ; the information is still reachable through the stock production lot record